CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 0.5.6
+
+* Update tax and benefit system initialization method in scripts.
```

Comment:
@MattiSG best answer: no impact. I had to produce a CHANGELOG message so did I. Now I'm completely sure in this case this would have been better not to release anything, and include these changes in the next release. Honestly I'm tired of this kind of things and the time it takes. A quick look at the commit shows that only scripts are modified, and the patch version number contains the information that it's quite nothing.